include/seqan3/range/views/translate.hpp

```diff
@@ -6,796 +6,14 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
+ * \brief [DEPRECATED] Provides seqan3::views::translate and seqan3::views::translate_single.
  * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
```

Comment:
I don't know if it is important, but usually we have author and then brief. Could you fix this?

Reply:
If I understand correctly, it's the other way around. So I will correct it with the other files.
https://github.com/seqan/seqan3/wiki/Writing-Documentation#annotating-general